tutorials/setting_up_CAD_models.md

```diff
@@ -0,0 +1,316 @@
+  * **Large Parts** (robot arms, major components): **\< 100,000 tris**  
+  * **Very Large Parts** (enclosures, machine frames): **\< 200,000 tris**
+
+### ---
+
+#### **Step 4: Export the Final Mesh 📤**
+
+Once your model is optimized and converted, you're ready to export it. The file format you choose matters.
+
+* **Export as a Mesh:** Use your CAD software’s `Export` or `Save As` function and choose a mesh file format.  
+* **Choose the Best Format:** Select the most ideal format that your CAD software supports, following this order of preference:  
+  1. **GLB (.glb):** **(Most Ideal)** This format is effective because it can bundle everything—the mesh, materials, and textures—into a single file.  
+  2. **FBX (.fbx):** A very common and robust format in the 3D industry that handles mesh and material data well.  
+  3. **OBJ (.obj):** A widely supported and reliable format, though it may handle materials less gracefully than GLB or FBX.  
+  4. **STL (.stl):** **(Least Ideal)** Use this only as a last resort. It typically contains only raw geometry data and no material or color information.
+
+Note at this point, if your object does not have any kinematics, you can directly import the object. If you want to further refine parts, we recommend following the  mesh refinement instructions  in Blender. 
+
+If you are satisfied with the mesh refinement and want to upload a file with Kinematics you may do so by authoring and SDF manually. See the [Creating an SDF manually](#section-4:-manual-sdf-file-creation) section
+
+If you would like to use the Blender plugin to automatically author the SDF,  follow the steps below.   
+---
+
+### **Section 3:  Using Blender for Mesh Optimization and SDF Generation**
+
+This section explains how to use the custom SDF gen plugin to author the SDF and perform additional mesh simplification steps if needed. 
+
+####  **Section 3: Part 1 \- Initial Setup & Model Preparation**
+
+Follow these steps to install the required addon and prepare your mesh geometry for the SDF generation process.
+
+##### **Step 1: Addition of the plugin to Blender⚙️**
+
+  Following steps show how to activate the SDF format add-on inside Blender.
+
+1. **Download the Repository:** First, you need to download the plugin and add it to Blender. The repository can be found here \- [https://github.com/cole-bsmr/SDF\_Gen](https://github.com/cole-bsmr/SDF_Gen) .  
+2. In Blender, **Open Preferences**: In the top menu bar, click `Edit` \> `Preferences`.  
+3. **Go to Add-ons**: In the new window, select the `Add-ons` section from the left-hand menu.  
+4. Click on **“Install from Disk”** and search for the file to upload it.   
+5. **Enable the Add-on**: The "Import-Export: SDF format" add-on will appear. Click the checkbox next to it to enable it.  
+6. **Close Preferences**: The change is saved automatically. You can now close the Preferences window.
+
+![][files/setting_up_CAD_models/install_blender_plugin.gif]
```

Comment:
As per the github [markdown syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) guide I think this line should be

```suggestion
![](files/setting_up_CAD_models/install_blender_plugin.gif)
```

Same for the images below.